### ISSUE TITLE
Theme component does not change the color of all encapsulated components

### ISSUE THREAD

**Issue by echarbeto** [tamagui/tamagui#1725]
### Current Behavior

When I encapsulate Label, Input, Button, and SizableText components with the Theme component, only the color of the Input and Button components changes.

### Expected Behavior

I would expect all components within the Theme component to change color when a certain condition (e.g., hasError) is true.

### Tamagui Version

```markdown
1.73.0
```

### Platform (Web, iOS, Android)

```markdown
Web and Android
IDK IOS
```

### Reproduction

```markdown
import { Theme, Label, Input, Button, SizableText } from 'tamagui';

function MyComponent() {
  const hasError = /* logic to determine if there is an error */;

  return (
    <Theme name={hasError ? "red" : null}>
      <Label>Label</Label>
      <Input />
      <Button>Button</Button>
      <SizableText>Text</SizableText>
    </Theme>
  );
}
```

### System Info

```markdown
npmPackages:
    @babel/core: ^7.20.0 => 7.22.20 
    @react-native-async-storage/async-storage: 1.18.2 => 1.18.2 
    @reduxjs/toolkit: ^1.9.5 => 1.9.5 
    @tamagui/animations-react-native: ^1.73.0 => 1.73.0 
    @tamagui/babel-plugin: ^1.73.0 => 1.73.0 
    @tamagui/config: ^1.73.0 => 1.73.0 
    @tamagui/font-inter: ^1.73.0 => 1.73.0 
    @tamagui/linear-gradient: ^1.73.0 => 1.73.0 
    @tamagui/lucide-icons: ^1.73.0 => 1.73.0 
    @tamagui/theme-base: ^1.73.0 => 1.73.0 
    @types/react: ~18.2.14 => 18.2.22 
    babel-plugin-transform-inline-environment-variables: ^0.4.4 => 0.4.4 
    eslint: ^8.49.0 => 8.49.0 
    eslint-config-universe: ^12.0.0 => 12.0.0 
    expo: ~49.0.11 => 49.0.11 
    expo-build-properties: ~0.8.3 => 0.8.3 
    expo-constants: ~14.4.2 => 14.4.2 
    expo-dev-client: ~2.4.10 => 2.4.10 
    expo-font: ~11.4.0 => 11.4.0 
    expo-linear-gradient: ~12.3.0 => 12.3.0 
    expo-linking: ~5.0.2 => 5.0.2 
    expo-network: ~5.4.0 => 5.4.0 
    expo-router: 2.0.8 => 2.0.8 
    expo-status-bar: ~1.6.0 => 1.6.0 
    formik: ^2.4.4 => 2.4.5 
    metro-config: ^0.79.1 => 0.79.1 
    react: 18.2.0 => 18.2.0 
    react-dom: 18.2.0 => 18.2.0 
    react-native: 0.72.5 => 0.72.5 
    react-native-gesture-handler: ~2.12.0 => 2.12.1 
    react-native-google-mobile-ads: ^12.1.2 => 12.1.2 
    react-native-reanimated: ~3.3.0 => 3.3.0 
    react-native-safe-area-context: 4.6.3 => 4.6.3 
    react-native-screens: ~3.22.0 => 3.22.1 
    react-native-svg: 13.9.0 => 13.9.0 
    react-native-web: ~0.19.8 => 0.19.9 
    react-redux: ^8.1.2 => 8.1.2 
    tamagui: ^1.73.0 => 1.73.0 
    typescript: ^5.1.3 => 5.2.2 
    yup: ^1.2.0 => 1.2.0
```

**Comment by natew** [comment]
need a repro, this is extensively tested in the repo and across the site and works

**Comment by gitteraz** [comment]
Hi @echarbeto Did you manage to solve this? Facing the same issue after upgrading to 1.74.0.

**Comment by natew** [comment]
There was a temporary regression in 1.74 fixed in latest patch release.

**Comment by echarbeto** [comment]
Hello guys, sorry for the delay.

I wanted to create a repository to reproduce the issue I reported. I created this repository https://github.com/echarbeto/tamagui_test and added two encapsulated themes for some components for easy testing.

I believe the result is as expected because after reading better the documentation, I understood that Label and Text need the attribute ‘theme’ to change color according to their encapsulated theme.

Thanks!

![localhost_8082_](https://github.com/tamagui/tamagui/assets/26139299/f34d0f79-9147-41ca-ac8c-48138a34b567)
